spec/support/database_cleaner.rb

```diff
@@ -1,18 +1,18 @@
-Rspec.configure do |config|
+RSpec.configure do |config|
   config.before(:suite) do
-    Database.Cleaner.clean_with :truncation, except %w(ar_internal_metadata)
+    DatabaseCleaner.clean_with :truncation#, except %w(ar_internal_metadata)
```

Comment:
This line caused an issue in a different PR and I wondered why you were using `except`. If we're getting rid, don't comment it out, just remove :)